client/Pages/EventTypeDetails/Help.elm

```diff
@@ -387,3 +387,71 @@ subscription =
     , newline
     , spec "#/subscriptions_get"
     ]
+
+
+audience : List (Html msg)
+audience =
+    [ text "Intended target audience of the event type."
+    , newline
+    , text "Relevant for standards around quality of design and documentation,"
+    , text " reviews, discoverability, changeability, and permission granting."
+    , newline
+    , newline
+    , link "See the guidelines" "https://opensource.zalando.com/restful-api-guidelines/#219"
+    , newline
+    , newline
+    , bold "Key: "
+    , mono "audience"
+    , newline
+    , bold "optional"
+    , newline
+    , man "#definition_EventType*audience"
+    ]
+
+
+cleanupPolicy : List (Html msg)
+cleanupPolicy =
+    [ text "Event type cleanup policy."
+    , newline
+    , bold "Possible cleanup policies are:"
+    , newline
+    , text "- "
+    , mono "delete"
+    , text " will delete old events after retention time expires."
+    , newline
+    , text "- "
+    , mono "compact"
```

Comment:
@SergKam could you please add also the following text in a very visible way:

"Log compacted event types **must not**  contain personal identifiable information in accordance to GDPR. If you plan to store user data permanently, check the legal department of your organization".

Reply:
![screenshot 35](https://user-images.githubusercontent.com/3240529/43313671-0c592c3e-9191-11e8-97f1-730d1ecd133a.png)

Reply:
@SergKam thank you. My mind also refuses to accept this, but it's when it comes to legal, we developer have a weaker hand on negotiations and discussions. If the guy on the other side has the authority in terms of security/legal, what can we do right? But I hope we can change it in the future once we define an API for deletion of events and the data classification process gets fully automated.